content/en/agent/autodiscovery/_index.md

```diff
@@ -129,19 +115,20 @@ ECS_FARGATE=true
 {{% /tab %}}
 {{< /tabs >}}
 
+### Integration Templates
+
+Once Autodiscovery is enabled, the Datadog Agent automatically attempts Autodiscovery [for a number of services][3], including Apache and Redis, based on default Autodiscovery configuration files.
+
+To use Autodiscovery with other services, define templates for the services you wish to monitor. See the [Autodiscovery Integration Templates][4] documentation for further details.
+
+
+
 ## Further Reading
```

Comment:
Can you add a section of the different checks that use/relate to Autodiscovery? Specifically I'd like to clarify when to use Cluster-checks, Agent-checks and others (i.e. end-point checks). We can also refer to the pages that explain them for further reading